content/docs/csidriver/installation/operator/unity.md

```diff
@@ -98,7 +98,7 @@ Users should configure the parameters in CR. The following table lists the prima
    | X_CSI_HEALTH_MONITOR_ENABLED                    | Enable/Disable health monitor of CSI volumes from Node plugin               | No       |                       |
 
 ### Example CR for Unity XT
-Refer samples from [here](https://github.com/dell/dell-csi-operator/tree/master/samples). Below is an example CR:
+Refer samples from [here](https://github.com/dell/csm-operator/blob/main/samples). Below is an example CR:
```

Comment:
This section is specific to dell-csi-operator. So please revert the change in this file
For driver installed via csm-operator, please add the required changes in https://dell.github.io/csm-docs/docs/deployment/csmoperator/drivers/unity/

Reply:
- done